libultraship/libultraship/Archive.cpp

```diff
@@ -68,7 +68,7 @@ namespace Ship {
 		//}
 
 		if (!attempt) {
-			printf("({%i}) Failed to open file {%s} from mpq archive {%s}", GetLastError(), filePath.c_str(), MainPath.c_str());
+			printf("(%i) Failed to open file { %s } from mpq archive { %s }\n", GetLastError(), filePath.c_str(), MainPath.c_str());
```

Comment:
We don't use printf in libultraship. Look for uses of SPDLOG_ macros to understand how we log in LUS.
https://github.com/gabime/spdlog